CONTRIBUTING.md

```diff
@@ -141,6 +141,7 @@
   - Use TailwindCSS (which are connected to design tokens) for all Solid components.
   - Convert BEM-style CSS to TailwindCSS utility classes where possible.
   - Utilize IDs or part selectors for any custom CSS needs.
+  - Use `@apply` inside `css` tagged template literals to generate CSS, but do not use arbitrary values like `mt-[var(--spacing-xxl)]` there (!), as this increases the bundle size of the main TailwindCSS file.
```

Comment:
is this important enough to highlight it a bit more or how big is the risk if people do it anyways?

Reply:
It might be helpful to mention what developers could do if they **need** to use variables. i.e "Consider adding custom values as plain CSS outside the @apply directive..."

Reply:
Done.